### ISSUE TITLE
Plate Reverb Bug: Iv'e got chocolate stains on my pants....

### ISSUE THREAD

**Issue by LKHSogpit** [nysthi/nysthi#24]
I have not been able to narrow this down yet to be honest I'd rather hang on to my ears for another few decades. Happened twice in two separate patches the Plate Reverb just gets a burst of noise this is no ordinary low dB sound it is so loud it masks every other sound. Had my volume about about 65-70dB and the sound I heard was probably about 90-100dB. First reaction was to press spacebar for some strange reason before reaching for the volume on the interface which the sound was still audible at lowest volume on the interface 0% both headphone and monitor volume were turned down all the way.

Still not over it actually typing with 1 hand and sucking my thumb with the other while I rock back and forth!

It may not be the plate reverb there was a lot of other stuff going on. I had the Plate Reverb connect up to a dBiz performance mixer send/return but the plate reverb ins/out were glowing white. It may have been something to do with the performance mixer both times the plate was connected to it. 

Is there any known issues with the plate reverb?

**Comment by LKHSogpit** [comment]
Also just happened with that Dissonant reverb with the patch that was submitted to the facebook group "SOU contrappunto"

**Comment by nysthi** [comment]
what is your system ? Windows ?

**Comment by nysthi** [comment]
after how many minutes ? was it immediate ?

**Comment by LKHSogpit** [comment]
Yes Windows 7. Patch was open for a good 3-4 minutes before the noise happened. I was navigating folders to open another patch.

**Comment by nysthi** [comment]
you are the fourth reporting the problem on windows
but I hope to reproduce it
if you have trick to reproduce at command, please let me know !!

**Comment by LKHSogpit** [comment]
It will be hard to nail down I think because it seems random. The patch then went on to play the audio again after I lowered the volume and put the Diss to fully dry. Using Asio driver and a focusrite 2i4 interface. Have you a link to the other reports or can you paste any info they have reported here I'll try narrow it down some more.

**Comment by nysthi** [comment]
anyway, when happens, just do INITIALIZE with the contextual menu

**Comment by LKHSogpit** [comment]
Have get new boxer shorts when it happens first priority!

**Comment by nysthi** [comment]
Any luck (bad luck...) reproducing it ? 
I have had a patch running for hours, with a lot  of transient and pops and noise and max volume and modulation
but the bug did not appear.

**Comment by LKHSogpit** [comment]
Same had it running for a good while last night with no joy. Even went as far as loading opera browser with 8gb of extensions and opening and closing repeatedly. Opera loads into ram and causes DSP spikes when opened. Just clicks and pops but no bug.

heisenbug...

**Comment by patman023** [comment]
[bass fart groove.zip](https://github.com/nysthi/nysthi/files/1992248/bass.fart.groove.zip)

Patch was broken by 0.6.0, as well as plugin name changes, but I've got it back to the point where it bugs out similar to OP. 

let it run for a while, it'll eventually happen. (it even happened once or twice to me while all mixers aside from master were muted.

**Comment by WouterSpekkink** [comment]
I was on here to report another issue, but I read this and realized that I have experienced the same problem. I did not even realize yet that it was the plateverb that causes this. For me it sometimes happens when I just loaded VCV rack (probably with a patch that includes the plateverb). It scared the shit out of me.

**Comment by patman023** [comment]
requesting bug/cant repro tags

**Comment by patman023** [comment]
Encouraging 1.x users to submit patches, as is still an issue. 

Please include OS (Including flavour/version number), Rack Version used for patch, Number of threads and Sample Rate (in Engine menu, NOT Audio Module), as well as your patch in a .txt format. 

Note that Firefox may prevent you from uploading, this is a Github issue.
